nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/operations/blob/BlobToFileTest.java

```diff
@@ -67,30 +67,33 @@
 
     protected Blob blob;
 
-    protected File pdfFile;
+    protected File directory = new File(DIRECTORY_PATH);
+
+    protected File pdfFile = new File(PDF_NAME);
 
     @Before
     public void setUp() throws Exception {
-        pdfFile = FileUtils.getFile(PDF_PATH);
+        IOUtils.copy(getClass().getClassLoader().getResourceAsStream(PDF_NAME), new FileOutputStream(pdfFile));
```

Comment:
`IOUtils.copy` does not close the streams, so you should write:

    try (InputStream in = getClass().getClassLoader().getResourceAsStream(PDF_NAME);
            OutputStream out = new FileOutputStream(pdfFile)) {
        IOUtils.copy(in, out);
    }